source/_components/fan.xiaomi_miio.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: page
+title: "Xiaomi Air Purifier 2"
+description: "Instructions how to integrate your Xiaomi Air Purifier 2 within Home Assistant."
+date: 2017-10-13 12:35
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: xiaomi.png
+ha_category: Switch
+ha_version: 0.56
+ha_iot_class: "Local Polling"
+---
+
+The `xiaomi_miio` fan platform allows you to control the Xiaomi Air Purifier 2. The Air Purifier Pro isn't supported right now.
+
+Currently, the supported features are
+
+* On, Off
+* Operation modes (auto, silent, favorite, idle)
+* Buzzer (on, off)
+* LED (on, off), LED brightness (bright, dim, off)
+* Favorite Level
+* States
+  - power
+  - aqi
+  - humidity
+  - temperature
+  - mode
+  - led
+  - led_brightness
+  - buzzer
+  - child_lock
+  - brightness
+  - favorite_level
+  - filter1_life
+  - f1_hour_used
+  - use_time
+  - motor1_speed
+
+## {% linkable_title Setup %}
+
+Please follow the instructions on [Retrieving the Access Token](/components/vacuum.xiaomi_miio/#retrieving-the-access-token) to get the API token to use in the `configuration.yaml` file.
+
+To add a Xiaomi Air Purifier to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+fan:
+  - platform: xiaomi_miio
+    name: Xiaomi Air Purifier 2
+    host: 192.168.130.66
+    token: YOUR_TOKEN
+```
+
+Configuration variables:
+- **host** (*Required*): The IP of your plug.
```

Comment:
Please use the new configuration variables jekyll tags. See DuckDNS for example https://github.com/home-assistant/home-assistant.github.io/blob/next/source/_components/duckdns.markdown